tests/io/feed_parsers/dpanewsml_test.py

```diff
@@ -106,6 +106,23 @@ def test_body_html(self):
 
             self.assertEqual(self.item.get("body_html").strip(), expected_output.strip())
 
+    def test_body_html_section_tags(self):
+        self.filename = "dpa_2.xml"
+        self.setUp()
+        self.maxDiff = None
+        with self.app.app_context():
+            self.assertIsInstance(self.item.get("body_html"), str)
+            expected_output = (
+                '<p><span class="dateline">BUENOS AIRES, Argentina <span class="credit">(AP)</span></span>'
```

Comment:
it might be good to also strip those spans while we're at it

Reply:
got it